library/std/src/fs/tests.rs

```diff
@@ -1534,3 +1534,14 @@ fn read_large_dir() {
         entry.unwrap();
     }
 }
+
+#[test]
+#[cfg(windows)]
+fn hiberfil_sys() {
```

Comment:
Maybe add a comment that this is testing the fallback method?

Reply:
Will do.